src/crimson/os/alienstore/alien_store.cc

```diff
@@ -619,9 +619,11 @@ std::vector<uint64_t> AlienStore::_parse_cpu_cores()
     std::string cpu;
     std::getline(cpu_stream, cpu, '-');
     uint64_t start_cpu = std::stoull(cpu);
+    uint64_t end_cpu = start_cpu;
     std::getline(cpu_stream, cpu, '-');
-    uint64_t end_cpu = std::stoull(cpu);
-    for (uint64_t i = start_cpu; i < end_cpu; i++) {
+    if(!cpu.empty())
+        end_cpu = std::stoull(cpu);
```

Comment:
```c++
if (!cpu.empty()) {
  end_cpu = std::stoull(cpu);
}
```

Reply:
@ljx023 the white space is not consistent.

Reply:
changed